docs/TEAMS.md

```diff
@@ -0,0 +1,109 @@
+Organic Maps is led by community contributors, with functional domains occasionally establishing or dissolving their leadership structures, formal or informal. 
+
+This file outlines the functional project structure that has evolved over time. The current list of teams is available on [GitHub](https://github.com/orgs/organicmaps/teams), accessible only to [GitHub Org members](https://github.com/orgs/organicmaps/people).
+
+Below is the list of teams and groups to tag on GitHub if you need assistance in a specific area. If you are unsure which team to contact, please tag  [@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage) for help.
+
+## Contributors
+
+[@organicmaps/contributors](https://github.com/orgs/organicmaps/teams/contributors/teams)
+
+Active code contributors who can create branches and manage issues in the main repo. See [subteams](https://github.com/orgs/organicmaps/teams/contributors/teams).
+
+### Triage
+
+[@organicmaps/triage](https://github.com/orgs/organicmaps/teams/contributors/triage)
+
+A special subteam to triage incoming issues and pull requests. Please tag this group if you're unsure which group to tag.
+
+## Maintainers
+
+[@organicmaps/maintainers](https://github.com/orgs/organicmaps/teams/maintainers)
+
+People who can merge PRs.
+
+## Android
+
+[@organicmaps/android](https://github.com/orgs/organicmaps/teams/android/members)
+
+Android contributors,
+
+### Android Auto
+
+Android Auto contributors (a subteam of Android).
+
+[@organicmaps/android-auto](https://github.com/orgs/organicmaps/teams/android-auto/members)
+
+## iOS
+
+[@organicmaps/ios](https://github.com/orgs/organicmaps/teams/core/members)
+
+iOS contributors.
+
+## C++
+
+[@organicmaps/cpp](https://github.com/orgs/organicmaps/teams/cpp/members)
+
+C++ contributors.
+
+## Data
+
+[@organicmaps/map](https://github.com/orgs/organicmaps/teams/data/members)
+
+Map data preparation and generation.
+
+## Styles
+
+[@organicmaps/styles](https://github.com/orgs/organicmaps/teams/styles/members)
+
+Map styles and visualization.
+
+## DevOps
+
+[@organicmaps/devops](https://github.com/orgs/organicmaps/teams/devops/members)
+
+DevOps group.
+
+## Design
+
+[@organicmaps/design](https://github.com/orgs/organicmaps/teams/design/members)
+
+Visual design (icons, graphics, colors, contrast, etc.).
```

Comment:
and UX?

Reply:
Visual Design and UI/UX are distinct disciplines, sharing only the word "design" in common. @oleg-rswll any thoughts? Do we need UI/UX team at all?

Reply:
Sure. We have their own resources even https://github.com/organicmaps/organicmaps/wiki/Design-Index

Reply:
A UX/UI team is likely needed, as a significant part of the app experience is the interface.  Graphic design can be included, although it's a more narrow discipline.  More broadly UI/UX is often part of the product team.  For now it could a separate team as people understand what it means.

Reply:
a Design Team with UI/UX, graphic design & product design subteams?

Reply:
Let's just keep it as "@design" for now?